src/app/components/search/SearchResultsCards/ArticleCard.js

```diff
@@ -51,21 +70,25 @@ const FactCheckCard = ({
   </div>
 );
 
-FactCheckCard.defaultProps = {
+ArticleCard.defaultProps = {
   summary: null,
   url: null,
   statusColor: 'black',
+  teamAvatar: null,
+  teamName: null,
+  variant: 'explainer',
 };
 
-FactCheckCard.propTypes = {
+ArticleCard.propTypes = {
   title: PropTypes.string.isRequired,
   statusLabel: PropTypes.string.isRequired,
   statusColor: PropTypes.string,
   date: PropTypes.number.isRequired, // Timestamp
   summary: PropTypes.string,
   url: PropTypes.string,
-  teamAvatar: PropTypes.string.isRequired, // URL
-  teamName: PropTypes.string.isRequired,
+  teamAvatar: PropTypes.string, // URL
+  teamName: PropTypes.string,
+  variant: PropTypes.oneOf(['explainer', 'fact-check']),
```

Comment:
There are some props missing to be declared here:

-   languageCode
-   tags
-   onChangeTags